source/administration/plugins.rst

```diff
@@ -12,12 +12,63 @@ Consider using a plugin in the following scenarios:
+ - **Server plugins**: Run a Go process alongside the server, filtering messages or integrating with third-party systems such as Jira, GitLab or Jenkins. For a sample plugin, see `our Jira plugin <https://github.com/mattermost/mattermost-plugin-jira>`__.
+
+Plugin Marketplace
+-------
+The Plugin Marketplace is a collection of plugins that can greatly increase the value and capabilities of your Mattermost deployment.  End-users can see increased productivity through quick access to systems such as Jira, Zoom, Webex, and GitHub.  Mattermost Administrators can discover new plugins and quickly deploy them to their servers, including High availability clusters.  It is available in 5.16 and is accessed via **Main Menu > Plugin Marketplace**.  Our Marketplace listings will continue to expand as we add new features and plugins.  
+
+.. image:: https://user-images.githubusercontent.com/915956/66891467-1b18eb80-ef9e-11e9-9de3-37a3c5899bd8.png
+
+Installing a Plugin
+~~~~~~~~~~~~~~~~
+
+When a new plugin becomes available on the marketplace, a new listing with a button labeled **Install** will be displayed.  After clicking **Install**, the latest plugin binary is downloaded from its respective source and installed on the server.    
+
+Configuring and Enabling a Plugin
+~~~~~~~~~~~~~~~
+
+Once a plugin has been installed (or pre-installed if it shipped with the Mattermost binary release) there will be a **"Configure"** button shown next to the plugin listing.  
+
+1. Click on the button and you will be navigated to the plugins **"Settings"** page.  
+2. Fill in Plugin settings as required.
+3. At the top of the page, set **Enable Plugin** setting to ``True``. The plugin will remain inactive until this flag is switched on.
+4. Test out the plugin as needed.
+
+Upgrading Plugins
+~~~~~~~~~~~~~~~~~
+
+In v5.16, the Marketplace only supports the installation of new plugins.  To upgrade a plugin, you will need to manually update it by downloading the binary file from the GitHub repository and uploading it in the **System Console > Plugin management** section.  In an upcoming release, the Marketplace will allow you to upgrade a plugin on demand when a new version becomes available.
+
+Marketplace Server
+-----
+
+There are two Marketplace settings in the **System Console > Plugin management** section:
+
+.. image:: https://user-images.githubusercontent.com/915956/66892854-94660d80-efa1-11e9-805c-a85223d43a07.png
+
+- **Enable Marketplace**: Turns the Marketplace User Interface on or off for System Administrators.
+- **Marketplace URL**: The location of the Marketplace Server you want to query for new plugins. Mattermost hosts a Plugin Marketplace for the community and is the default value for this field. You can setup your own Marketplace server if you want to rely on your own infrastructure.
```

Comment:
@aaronrothschild the other paragraph that was removed mentioned the marketplace server repo, should that be included here to guide admins to the repo?
https://github.com/mattermost/docs/pull/3073/commits/5569cad34733a1ca162f17b4e2e7eb3d0a94efd5
```
The `Plugin marketplace server code <https://github.com/mattermost/mattermost-marketplace>`__ is available as an open-source project and can be used to setup your own private Marketplace if desired.
```

Reply:
It's included in a paragraph below, didn't need it duplicated..